### DISCUSSION TITLE
Window not display in `macOS` with `11.0.0 Preview 6`

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#10761] [Q&A] [answered]
![PC_BB3F32MY$GXNH%8_ UGN](https://user-images.githubusercontent.com/26276037/227182164-3135c0a1-11f5-4929-a017-2ed120cd7f88.png)
windows and linux is display.
![)7BT@$YZ$E EEG Y`LV(3~X](https://user-images.githubusercontent.com/26276037/227182257-560e7bf4-ebe0-4952-8076-b02d3338a664.png)
![G@ H%5AS)$_BAR2M00Y`W](https://user-images.githubusercontent.com/26276037/227182359-f253916a-0412-4657-ab16-274d886f296f.png)

**Comment by Coloryr** [comment]
![HW{RSI9I$D%6EAZQF3$HY9](https://user-images.githubusercontent.com/26276037/227182464-68967fec-b0da-4f62-9819-a3b16354d4dc.png)

**Comment by timunie** [comment]
Honestly, posting 2 images is not a good issue description. What needs to be done to reproduce the issue? Can you post a minimal sample?

**Reply by Coloryr** [reply]
![8CB2HI8M{V$GE)D AY1N~QW](https://user-images.githubusercontent.com/26276037/227201353-cf0c2480-d9b9-4547-acc9-efd1c4b7634c.png)
[AvaloniaApplication1.zip](https://github.com/AvaloniaUI/Avalonia/files/11050676/AvaloniaApplication1.zip)

**Reply by timunie** [reply]
looks like you have a refence which may not be up to date with preview 6. does it work if you remove this package? 

```diff
  <ItemGroup>
	  <PackageReference Include="Avalonia" Version="11.0.0-preview6" />
-	  <PackageReference Include="Avalonia.AvaloniaEdit" Version="11.0.0-preview5" />
	  <PackageReference Include="Avalonia.Controls.ColorPicker" Version="11.0.0-preview6" />
	  <PackageReference Include="Avalonia.Controls.DataGrid" Version="11.0.0-preview6" />
	  <PackageReference Include="Avalonia.Desktop" Version="11.0.0-preview6" />
	  <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.0-preview6" />
	  <PackageReference Include="Avalonia.Themes.Simple" Version="11.0.0-preview6" />
	  <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.0-preview6" />
	  <PackageReference Include="Avalonia.Markup.Xaml.Loader" Version="11.0.0-preview6" />
  </ItemGroup>
```

**Reply by Coloryr** [reply]
no work

**Reply by Coloryr** [reply]
![{C% 3_@)2QY`2XY IAA8({3](https://user-images.githubusercontent.com/26276037/227240196-0e6175ce-9b04-4a17-8548-95490de2bb14.png)
![7XS1XZ9 6OB6{91QGRZZ{KW](https://user-images.githubusercontent.com/26276037/227240323-9aa1b48c-3b18-437c-a66e-4f2c4e91c096.png)

**Reply by timunie** [reply]
@Coloryr I'm sorry, but we are not able to reproduce the issue. I've ask someone with a mac to test and he was able to start the App. 

See: 

![image](https://user-images.githubusercontent.com/47110241/227243152-2c898291-5c7d-4e94-ace8-750505f4bff6.png)

**Reply by Coloryr** [reply]
may work in higher version macOs, i test in `macos12.1`, but in `v11p4`, it can display.

**Reply by Coloryr** [reply]
![3 K PP4}}WA~B4UY%L)` 1](https://user-images.githubusercontent.com/26276037/227495619-66294897-93e6-4c3a-8d1b-4c81ea6b6cd8.png)
![4HS4BSEGU02CZ6%) T9EPIY](https://user-images.githubusercontent.com/26276037/227495718-aa3200f5-3b16-4e28-b0b3-d11b7f54713b.png)

**Reply by Coloryr** [reply]
![JR}(GPPOQBJ0F)F)M2M%A8K](https://user-images.githubusercontent.com/26276037/227495888-70bfae9a-51cc-4fb3-b1f7-f74d9855b8bd.png)
![T14(7P_7NYDT`M9 L ~Z6C1](https://user-images.githubusercontent.com/26276037/227496044-8749f3b8-3671-4203-be89-580d64fa0cf2.png)

it look like `v11p5` problem

**Reply by timunie** [reply]
Are you running it in a virtual machine on a windows device?

**Reply by Coloryr** [reply]
yes

**Reply by timunie** [reply, marked as answer]
I think that is not supported. See https://github.com/AvaloniaUI/Avalonia/issues/10770#issuecomment-1481737052

> We currently don't support running on non-Apple hardware.
> Also not that running macOS on non-Apple hardware isn't exactly legal.
